lib/src/insertion_order_parser.dart

```diff
@@ -166,4 +160,14 @@ class InsertionOrderParser {
         InsertionOrder_Budget_InsertionOrderAutomationType
             .INSERTION_ORDER_AUTOMATION_TYPE_UNSPECIFIED;
   }
+
+  static bool _isActiveSegment(
+      InsertionOrder_Budget_BudgetSegment segment, DateTime now) {
+    final startDate = segment.dateRange.startDate;
```

Comment:
On the api doc: The start and end date settings of the budget segment. They are resolved relative to the parent advertiser's time zone.

Need to get the advertiser time zone from public api. 
Issue: #51